Scripts/New-InitialDotNetCoreProjects.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+.SYNOPSIS
+Create an initial dotnet core project
+
+.DESCRIPTION
+Create an initial dotnet core project,  also populating the project guids from the solution file into the csproj files.
+
+.PARAMETER Prefix
+The prefix for the project name.  
+The script will create:
+
+A web project called $Prefix
+A unit test project called $Prefix.UnitTests
+A solution file called $Prefix.sln
+
+.EXAMPLE
+New-InitialDotNetCoreProjects.ps1 --Prefix SomeProject
+
+#>
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $Prefix
+)
+
+function New-BasicProject {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $Name    
+    )
+
+    $testProject = $Name + ".UnitTests"
+
+    & "dotnet" "new" "web" "--name" $Name
```

Comment:
I think we need to account for different project types, ie function apps

Reply:
I've added the ability to specify if you want a web or function project to the script.
We can add more types in later should we need to.